packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3215,16 +3218,26 @@ export class ContainerRuntime
 		);
 		if (recoveryMethod === "restart") {
 			const error = new GenericError("Restarting summarizer instead of refreshing");
+
+			const closeSummarizerDelayOverrideMs =
+				this.mc.config.getNumber(
+					"Fluid.ContainerRuntime.Test.CloseSummarizerOnSummaryStaleDelayOverrideMs",
+				) ?? defaultCloseSummarizerDelayMs;
```

Comment:
It would be great to open a bug like 2 sprints out to review telemetry and start cleaning code. I.e. if data suggests we do not need flights here, or delays - we should clean them

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/4109